src/Shared/CommunicationsUtilities.cs

```diff
@@ -465,7 +465,7 @@ internal static async Task<int> ReadAsync(Stream stream, byte[] buffer, int byte
             int totalBytesRead = 0;
             while (totalBytesRead < bytesToRead)
             {
-                int bytesRead = await stream.ReadAsync(buffer.AsMemory(totalBytesRead, bytesToRead - totalBytesRead), CancellationToken.None).ConfigureAwait(false);
+                int bytesRead = await stream.ReadAsync(buffer.AsMemory(totalBytesRead, bytesToRead - totalBytesRead), CancellationToken.None).ConfigureAwait(true);
```

Comment:
The change to true here was done because the file is used as apart of MSBuild. So its not apart of "library code". Additionally If you follow the calls up its executed on its own thread.